### PR TITLE
DOCS: Update Simplified Chinese translations

### PR DESCRIPTION
**Pull request by ice-yu**

### CODE REVIEW

File: psychopy/app/locale/zh_CN/LC_MESSAGE/messages.po [hunk 1]
```diff
@@ -30,114 +30,131 @@ msgstr ""
...
 "has been used incorrectly."
 msgstr ""
+"Builder无法解释{component.type}组件的“{param.val}”值\n"
+"“{component.params[name].val}”被误用为美元符号（$）。"
 
 #: ../alerts/alertsCatalogue/alertmsg.py:67
 msgid ""
 "Font `{param.val}` not found locally, will check Google Fonts on next run."
-msgstr ""
+msgstr "为在本地找到“{param.val}”字体，下次运行将在Google Fonts中查找。"
```
Comment (shun2wang) [inline psychopy/app/locale/zh_CN/LC_MESSAGE/messages.po]:
```suggestion
msgstr "未在本地找到“{param.val}”字体，下次运行将在Google Fonts中查找。"
```

Great work! just a typo:-)
You can just click the `Commit suggestion` below to add this change.

Comment (ice-yu) [reply psychopy/app/locale/zh_CN/LC_MESSAGE/messages.po]: Thanks!

**Review by shun2wang — APPROVED** [review]

**Comment by shun2wang** [comment]
If you want to using force-push you should always using `git fetch --all` first and then using `git rebase -i` to resolve conflicts then you can `git push -f` shoud not auto close this.

**Comment by shun2wang** [comment]
you can make a new one in this situation.

**Pull request closed without merging**